modules/ROOT/pages/mtf-expecting-errors.adoc

```diff
@@ -0,0 +1,75 @@
+= Expected Errors
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Validate the error types and error descriptions that your module throws.
+
+== Before You Begin
+
+* You must know how to define the errors that your module throws. For more information, refer to xref:errors.adoc[Errors Definition].
+
+== Example
+
+Suppose there is a module that defines errors as follows:
```

Comment:
I sort of want more context here - why are you giving this example (to use later?). Is there any more info you can provide about the examples?

Reply:
(I don't think we have any guidance on "Suppose...")

Avoid "there is" construction, if you can. 

You could say something like, In this example, the module defines.... The following sections describe...